benchmark/micro/aggregate/grouped_distinct.benchmark

```diff
@@ -7,6 +7,7 @@ group aggregate
 
 load
 CREATE TABLE integers AS SELECT i % 5 AS i, i % 25 as j FROM range(0, 10000000) tbl(i);
+set threads=1;
```

Comment:
Limiting threads seems like the wrong fix since the purpose of the benchmark runner is to measure performance - can we add an ORDER BY instead?